### ISSUE TITLE
Op request: layer normalization

### ISSUE THREAD

**Issue by vk1214** [tensorflow/tensorflow#60532]
Layer normalization is available in tensorflow https://www.tensorflow.org/api_docs/python/tf/keras/layers/LayerNormalization
It is not part of the tflite supported ops
https://github.com/tensorflow/tensorflow/blob/master/tensorflow/lite/builtin_ops.h

I would like to request support for this op. Please let me know if it is in development or scoped for future release.

**Comment by apage224** [comment]
Bump ^

**Comment by jens-totemic** [comment]
We are also interested in this. It appears that exporting a model that uses `LayerNormalization` will disable the TfLite XNNPack delegate, thus reducing performance of our model by a lot. 

Details:
After exporting the model, one of the layer norm tensors has `allocation_type==kTfLiteDynamic`. Since TfLiteXNNPackDelegate does not support dynamically allocated tensors (`delegate->flags & kTfLiteDelegateFlagsAllowDynamicTensors == 0`), the delegate is disabled and TF lite switches to using the default backend.

**Comment by NathanielB123** [comment]
I also would be very interested in this. While layer norms can be decomposed into multiple operators (`Mean`, `SquaredDifference`, `Rsqrt` etc..), this becomes problematic when doing quantisation (what operators should a quantised layer norm be converted into?)

**Comment by karkapur** [comment]
@NathanielB123 could you show what does your layer norm block look after quantisation ( Netron view maybe)? I'm getting weird connections after quantising with no similarity in to the actual code.TIA

**Comment by gaikwadrahul8** [comment]
Hi, @vk1214 

Thanks for raising this issue. Are you aware of the migration to [LiteRT](https://developers.googleblog.com/en/tensorflow-lite-is-now-litert/)? This transition is aimed at enhancing our project's capabilities and providing improved support and focus for our users. As we believe this issue is still relevant to LiteRT we are moving your issue there. Please follow progress here: https://github.com/google-ai-edge/LiteRT/issues/173

Let us know if you have any questions. Thanks.
